nvflare/app_common/abstract/fl_model.py

```diff
@@ -11,84 +11,83 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 from enum import Enum
-from typing import Dict, Optional
+from typing import Any, Dict, Optional
 
 
-class TransferType(Enum):
-    MODEL = "MODEL"
-    MODEL_DIFF = "MODEL_DIFF"
-    METRICS_ONLY = "METRICS_ONLY"
+class ParamsType(str, Enum):
+    WEIGHTS = "WEIGHTS"
```

Comment:
It's a bit weird to have WEIGHTS here. Maybe better just use `FULL` and `DIFF`.

Reply:
We only support WEIGHTS and WEIGHT_DIFF as of now, we can improve this later.